Open Judge System/Workers/OJS.Workers.ExecutionStrategies/NodeJsExecuteAndRunAsyncJsDomTestsWithReactExecutionStrategy.cs

```diff
@@ -41,27 +41,27 @@ public class NodeJsExecuteAndRunAsyncJsDomTestsWithReactExecutionStrategy :
         {
             if (!File.Exists(sinonJsDomModulePath))
             {
-                throw new ArgumentException($"SinonPackaged not found in: {sinonJsDomModulePath}", nameof(sinonJsDomModulePath));
+                throw new ArgumentException($@"SinonPackaged not found in: {sinonJsDomModulePath}", nameof(sinonJsDomModulePath));
```

Comment:
I checked it more thoroughly and I was wrong, it seems Resharper wants me to export the string literals into resource files.